File: 03-RomanNumerals/csharp-dotnetframework/RomanNumeralsTest/IntegerToRomanConverterTest.cs
using NUnit.Framework;
using RomanNumerals;

namespace Katas
{
    [TestFixture]
    public class IntegerToRomanConverterTest
    {
        [Test]
        [Ignore("Remove this line when you're ready to run the tests")]
        public void ShouldConvert16ToXVI()
        {
            var converter = new IntegerToRomanConverter();

            var actualResult = converter.Convert(16);
            var expectedResult = "XVI";

            Assert.AreEqual(expectedResult, actualResult);
        }
    }
}

File: 03-RomanNumerals/java/src/test/java/IntegerToRomanConverterTest.java
import static org.junit.Assert.assertEquals;

import org.junit.Ignore;
import org.junit.Test;

public class IntegerToRomanConverterTest {

  @Test
  @Ignore("Remove this line when you are ready to run this test")
  public void convertsSixteenToXVI() throws Exception {
    IntegerToRomanConverter converter = new IntegerToRomanConverter();

    assertEquals("XVI", converter.convertToInteger(16));
  }

}
